ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -136,23 +133,30 @@ Verify User Can Create And Start A Workbench With Existent PV Storage
 
 Verify User Can Create A PV Storage
     [Tags]    Sanity    Tier1    ODS-1819
-    ...       AutomationBug
     [Documentation]    Verifies users can Create PersistentVolume Storage
     ...                THIS MUST BE UPDATED TO CHECK WORKBENCH GETS RESTARTED LIKE FOR ODS-1825.
     ...                MAIN GOAL OF THE TEST CASE IS COVERED BY ODS-1814
     ${pv_name}=    Set Variable    ${PV_BASENAME}-A
     ${ns_name}=    Get Openshift Namespace From Data Science Project   project_title=${PRJ_TITLE}
     Open Data Science Project Details Page       project_title=${PRJ_TITLE}
-    ${workbenches}=    Create Dictionary    ${WORKBENCH_TITLE}=mount-data
-    Create PersistentVolume Storage    name=${pv_name}    description=${PV_DESCRIPTION}
-    ...                               size=${PV_SIZE}    connected_workbench=${NONE}     press_cancel=${TRUE}
-    ...                               project_title=${PRJ_TITLE}
-    Create PersistentVolume Storage    name=${pv_name}    description=${PV_DESCRIPTION}
-    ...                               size=${PV_SIZE}    connected_workbench=${workbenches}   project_title=${PRJ_TITLE}
+    Create Workbench    workbench_title=${WORKBENCH_TITLE}  workbench_description=${WORKBENCH_DESCRIPTION}
```

Comment:
what it the benefit of creating the workbench here? 
Unless you check the workbench gets restarted when the PVC gets created and attached, it is not needed